openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -1102,25 +1102,29 @@ def _elf_compute_electrostatic_energy(
         ).reshape(-1, 1)
 
         # Build an exclusion list for 1-2 and 1-3 interactions.
-        excluded_pairs = {
-            *[(bond.atom1_index, bond.atom2_index) for bond in molecule.bonds],
-            *[
-                (angle[0].molecule_atom_index, angle[-1].molecule_atom_index)
-                for angle in molecule.angles
-            ],
-        }
+        excluded_x, excluded_y = zip(
+            *{
+                *[(bond.atom1_index, bond.atom2_index) for bond in molecule.bonds],
+                *[
+                    (angle[0].molecule_atom_index, angle[-1].molecule_atom_index)
+                    for angle in molecule.angles
+                ],
+            }
+        )
 
         # Build the distance matrix between all pairs of atoms.
         coordinates = conformer.value_in_unit(unit.angstrom)
 
-        distances = np.sqrt(
-            np.sum(np.square(coordinates)[:, np.newaxis, :], axis=2)
-            - 2 * coordinates.dot(coordinates.T)
-            + np.sum(np.square(coordinates), axis=1)
-        )
+        # (a - b)^2 = a^2 - 2ab + b^2
+        ab = coordinates.dot(coordinates.T)
```

Comment:
```suggestion
        # (a - b)^2 = a^2 - 2ab + b^2
        distances_squared = coordinates.dot(coordinates.T)
```

Picky, I know, but I wasn't sure what `ab` is supposed to be until I got to the bottom

Reply:
Done in [`2a58f23` (#1070)](https://github.com/openforcefield/openff-toolkit/pull/1070/commits/2a58f23b891703cfe46630e7f30cbb414c6a7324) !